desktop/install/mac-install.md

```diff
@@ -101,6 +101,8 @@ $ sudo /Volumes/Docker/Docker.app/Contents/MacOS/install
 $ sudo hdiutil detach /Volumes/Docker
 ```
 
+The `install` command can take several minutes to run, because macOS typically performs security checks the first time an application is used.
+
 The `install` command accepts the following flags:
```

Comment:
Not super happy with the repetition here, @aevesdocker if you have a suggestion?

Reply:
My suggestion above would get rid of the repetition

Reply:
All right, if you’re happy with then so am I 😄